code/game/objects/items/rcd/RCD.dm

```diff
@@ -65,7 +65,7 @@
 	var/list/design = GLOB.rcd_designs[root_category][design_category][1]
 
 	rcd_design_path = design["[RCD_DESIGN_PATH]"]
-	design_title = initial(rcd_design_path.name)
+	design_title = initial(rcd_design_path.rcd_spritesheet_override) || initial(rcd_design_path.name)
```

Comment:
should we just have a getter for this/a helper proc?

Reply:
fixed